index.html

```diff
@@ -72,6 +72,14 @@ <h1 class='f3 fw2 montserrat aqua ttu'>CID Inspector</h1>
       </label>
       <div class='f5 sans-serif ma0 fw4 pt2'id="multihash"></div>
     </div>
+    <div class='pt4'>
+      <label class='dib'>
+        <a class='db pb1 w-100 tracked ttu f6 teal-muted hover-aqua link' href="https://docs.ipfs.tech/concepts/content-addressing/#version-1-v1">
+          CID length (Bytes)
```

Comment:
```suggestion
          CID length (raw bytes, without multibase envelope)
```

Reply:
So the byte length of the CID is of the binary representation that doesn't include the multibase part? How many bits/bytes is that?

Seems like it might be useful to show the byte length of the string representation

Reply:
@lidel Thanks for the suggestion,

I ended up rendering 3 values for the byte length:
- input
- base32 (only if the input is not base32)
- binary

Let me know what you think.

Examples:
- [base256]( https://bafybeicfp2qhnnzt6oblyy6kissn2dtaejifnq6ieo4k6ijxjv46lu4jx4.on.fleek.co/#%F0%9F%9A%80%F0%9F%AA%90%F0%9F%A5%BA%F0%9F%9A%80%F0%9F%A5%B0%F0%9F%8C%94%F0%9F%AA%90%F0%9F%92%BB%F0%9F%98%85%F0%9F%98%B9%E2%96%B6%F0%9F%8D%80%F0%9F%98%B0%F0%9F%91%8F%F0%9F%92%A2%F0%9F%91%8D%F0%9F%8C%BB%F0%9F%98%B6%F0%9F%8C%8F%F0%9F%8C%98%F0%9F%A5%82%F0%9F%98%9B%F0%9F%98%89%F0%9F%92%A6%E2%98%80%F0%9F%92%97%E2%9D%8C%F0%9F%A5%80%E2%9E%A1%F0%9F%92%95%F0%9F%91%85%F0%9F%98%91%F0%9F%98%93%F0%9F%A4%94%F0%9F%91%86%E2%9D%A4%F0%9F%92%AB%F0%9F%98%9A%F0%9F%A4%A3%F0%9F%98%B4%E2%9C%8A)
- [base36](https://bafybeicfp2qhnnzt6oblyy6kissn2dtaejifnq6ieo4k6ijxjv46lu4jx4.on.fleek.co/#k51qzi5uqu5djlam2gttbh69lponwxzjxd72nj27mdzkn1lrnd7ppvu9m3hdbc)
- [base32]( https://bafybeicfp2qhnnzt6oblyy6kissn2dtaejifnq6ieo4k6ijxjv46lu4jx4.on.fleek.co/#bafzaajaiaejcbcgbnsvcdmi7qowa4dh7r4wxmekbu7w4ehhpk5xstyyxrjtbs2fi)